.github/workflows/ci.yml

```diff
@@ -19,12 +19,15 @@ jobs:
     - name: module api test
       run: ./runtest-moduleapi --verbose
 
-  build-ubuntu-old:
-    runs-on: ubuntu-16.04
+  build-debian-old:
+    runs-on: ubuntu-latest
+    container: debian:oldoldstable
```

Comment:
i see now we are getting 404 on oldoldstable:
https://github.com/redis/redis/actions/runs/4794741534/jobs/8528460911
```
E: Failed to fetch http://security.debian.org/debian-security/dists/oldoldstable/updates/main/binary-amd64/Packages  404  Not Found [IP: 151.101.2.132 80]
E: Failed to fetch http://deb.debian.org/debian/dists/oldoldstable/main/binary-amd64/Packages  404  Not Found
E: Failed to fetch http://deb.debian.org/debian/dists/oldoldstable-updates/main/binary-amd64/Packages  404  Not Found
```

should we remove the job? or use another container